src/config/dynamic-theme-fixes.config

```
@@ -2919,6 +2919,13 @@ CSS
 
 ================================
 
+dawn.com
+
+INVERT
+div.mb-3 > a > img
```

```suggestion
img[alt="Dawn Logo"]
```

Using this selector makes it a little bit clearer what the CSS means. Thereby the `div.mb-3` seems to be the class for the header on every page and thus could result in other images being inverted.

---

Thanks for correcting me, I am still a beginner to css. Maybe I should have created an issue instead...